stringdtype/stringdtype/src/dtype.c

```diff
@@ -179,13 +179,32 @@ nonzero(void *data, void *NPY_UNUSED(arr))
 // Implementation of PyArray_CompareFunc.
 // Compares unicode strings by their code points.
 int
-compare_strings(char **a, char **b, PyArrayObject *NPY_UNUSED(arr))
+compare(void *a, void *b, void *NPY_UNUSED(arr))
 {
     ss *ss_a = (ss *)a;
     ss *ss_b = (ss *)b;
     return strcmp(ss_a->buf, ss_b->buf);
 }
 
+// PyArray_ArgmaxFunc
```

Comment:
`PyArray_ArgmaxFunc` isn't a thing, there's a `PyArray_ArgFunc` typedef for both argmin and argmax though.